simulator/carla_autoware/README.md

```diff
@@ -127,7 +127,6 @@ The `carla_ros.py` sets up the CARLA world:
 ## Tips
 
 - Misalignment might occurs during initialization, pressing `init by gnss` button should fix it.
-- Changing the `fixed_delta_seconds` can increase the simulation tick (default 0.05 s), some sensors params in `objects.json` need to be adjusted when it is changed (example: LIDAR rotation frequency have to match the FPS).
```

Comment:
After [this commit](https://github.com/mraditya01/autoware.universe/commit/84f286d845bb23769e55dfc0cb564c844162df2c), it seems we should fix the `rotation_frequency`.

Reply:
I think this tip should not be removed from the README.

Reply:
Thank you for the comment!
I agree to your point. I won't remove it.